data/ourCrewData.js

```diff
@@ -0,0 +1,35 @@
+const OurCrewData = [
```

Comment:
A tiny thing.
Usually we name these varaibles using the snakeCase.
```suggestion
const ourCrewData = [
```

Reply:
Only the React components (or classes) needs to start with a Capital letter.